python/PiFinder/server.py

```diff
@@ -983,6 +1122,43 @@ def time_lock(time=datetime.now()):
     def key_callback(self, key):
         self.keyboard_queue.put(key)
 
+    def _navigate_to_menu_path(self, menu_path):
+        """
+        Navigate to a specific menu path by simulating key presses
+        Path format: "settings/display" or "equipment/eyepiece"
+        """
+        if not menu_path:
+            return
+            
+        # This is a simplified navigation approach
+        # In practice, you might need more sophisticated logic
+        # depending on your menu structure and navigation patterns
+        
+        path_parts = menu_path.split('/')
+        logger.debug(f"Navigating to menu path: {path_parts}")
+        
+        # First, go to main menu (simulate square key to go back to root)
+        for _ in range(5):  # Max depth to prevent infinite loop
+            self.key_callback(self.ki.SQUARE)
+            time.sleep(0.05)
+        
+        # Now navigate through each part of the path
+        # This is a basic implementation - you may need to adjust
+        # based on your specific menu structure
+        for part in path_parts:
+            if part.strip():
```

Comment:
Have you tested this?  Looks like a partial implementation here :-)

Reply:
just saw this remark, you are correct, I'll retest to make sure everything works. The text-help kinda depends on this so will do this first